### PR TITLE
refactor(perception/lidar_apollo_instance_segmentation): rework parameters

### PR DESCRIPTION
**Pull request by karishma1911**
## Description

Implement the ROS Node configuration layout described in https://github.com/orgs/autowarefoundation/discussions/3371 for the perception_apollo_instance_segmentation

1. Remove the default value from the source code in order to ensure all parameter values are passed from the parameter files.

2.create schema

## Tests performed

Not applicable.

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by yukke42 — APPROVED** [review]
I confirmed it works.

File: perception/lidar_apollo_instance_segmentation/schema/lidar_apollo_instance_segmentation.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+          "default": "base_link",
+          "description": "Pointcloud data is transformed into this frame."
+        },
+        "z_offset": {
+          "type": "number",
+          "default": -2.0,
+          "description": "z offset from target frame. [m]."
+        }
+      },
+      "required": [
```
Comment (miursh) [inline perception/lidar_apollo_instance_segmentation/schema/lidar_apollo_instance_segmentation.json:54]: [Question]
How did you decide on these required parameters?
Is there a reason you didn't include "onnx_file"?

Comment (karishma1911) [reply perception/lidar_apollo_instance_segmentation/schema/lidar_apollo_instance_segmentation.json:54]: Based on, from the file detector.cpp.

**Comment by idorobotics** [comment]
@yukkysaito please review and merge?

**Comment by xmfcx** [comment]
@karishma1911 could you rebase this to the latest main? To fix the CI.

[Or click **Allow edits and access to secrets by maintainers.**](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork) For this PR. So we can do it for you.

**Review by yukkysaito — APPROVED** [review]
LGTM
cc @kminoda

**Comment by idorobotics** [comment]
> LGTM cc @kminoda

Thanks!

**Comment by miursh** [comment]
@idorobotics 
Could you please merge main branch to your branch again? In order to pass DCO check.

**Comment by idorobotics** [comment]
> @idorobotics 
> Could you please merge main branch to your branch again? In order to pass DCO check.

@karishma1911

**Comment by kminoda** [comment]
@karishma1911 Hi, thank you for your contribution!
To maintain the opened PR in Autoware Universe, let me convert this PR into a draft for now :pray: 
We are happy to review it again after the PR is fixed. 

Thank you for your understanding.

**Pull request closed without merging**